src/hooks/use-import-export.tsx

```diff
@@ -305,6 +306,7 @@ export const ImportExportProvider = ( { children }: { children: React.ReactNode
 					plugins: false,
 					themes: false,
 				},
+				phpVersion: selectedSite.phpVersion,
```

Comment:
The site export only produces the SQL file, so currently we are not exporting the PHP version in this case.

I presume the PHP version (as well as the WP version) are core dependencies that should be always synced when importing, but I wonder if the PHP version should be updated when only exporting the database 🤔.

Reply:
I would leave that as is - if users export SQL dump, they will be responsible for importing it to a correct PHP and WP version.